sirepo/package_data/static/js/sirepo.js

```diff
@@ -52,6 +52,9 @@ SIREPO.beamlineItemLogic = function(name, init) {
     });
 };
 
+SIREPO.BLOCKS = {};
```

Comment:
Not sure I like this name. Maybe SNIPPETS?

Reply:
4787fb9